service/src/main/java/uk/gov/hmcts/reform/fpl/utils/EmailNotificationHelper.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.hmcts.reform.fpl.utils;
+
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.Respondent;
+import uk.gov.hmcts.reform.fpl.model.RespondentParty;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+
+import java.util.Objects;
+
+import static org.apache.commons.lang3.StringUtils.isNoneBlank;
+import static org.apache.commons.lang3.StringUtils.isNotBlank;
+import static org.springframework.util.CollectionUtils.isEmpty;
+
+public class EmailNotificationHelper {
+
+    private EmailNotificationHelper() {
+    }
+
+    public static String buildSubjectLine(final CaseData caseData) {
+        final String lastName = (isNotBlank(getRespondents1Lastname(caseData))
+            ? String.format("%1$s", getRespondents1Lastname(caseData)) : "");
+
+        final String familyManCaseNumber = (isNotBlank(caseData.getFamilyManCaseNumber())
+            ? String.format("%1$s", caseData.getFamilyManCaseNumber()) : "");
+
+        return String.format("%1$s%2$s%3$s", lastName,
+            (isNoneBlank(lastName, familyManCaseNumber) ? ", " : ""),
+            familyManCaseNumber);
+    }
+
+    private static String getRespondents1Lastname(final CaseData caseData) {
+        return isEmpty(caseData.getRespondents1()) ? "" : caseData.getRespondents1()
```

Comment:
looking at this method, you're actually dealing with the null check here, so either you can edit your above methods to set lastName = this method, which then returns the last name or an empty string back. OR you can use defaultIfNull and have this method return null